filebeat/fileset/factory.go

```diff
@@ -179,8 +179,10 @@ func (p *inputsRunner) Start() {
 	}
 
 	// Loop through and add modules, only 1 normally
```

Comment:
Do you mind updating this old comment? The "only 1 normally" is not true anymore.